src/sql/azdata.proposed.d.ts

```diff
@@ -403,6 +403,25 @@ declare module 'azdata' {
 		title: string;
 	}
 
+	export interface ConnectionProvider extends DataProvider {
+		/**
+		 * Changes the password for a connection profile (for Azure Data Studio use only)
+		 */
+		changePassword?(connectionUri: string, connectionInfo: ConnectionInfo, newPassword: string): Thenable<PasswordChangeResult>;
```

Comment:
The comment is not accurate, it is not changing the password for a connection profile, this is only for the scenario of password expiration for SQL Authentication.